voter-dapp/src/ActiveRequests.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { drizzleReactHooks } from "drizzle-react";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+
+// TODO: Share this utility function with sponsor-dapp. React disallows imports outside of src/, so we'll need to find
+// some clever workaround.
+function formatDate(timestampInSeconds, web3) {
+  return new Date(
+    parseInt(
+      web3.utils
+        .toBN(timestampInSeconds)
+        .muln(1000)
+        .toString(),
+      10
+    )
+  ).toString();
+}
+
+function ActiveRequests() {
+  const { drizzle, useCacheCall } = drizzleReactHooks.useDrizzle();
+  drizzleReactHooks.useDrizzleState(drizzleState => {
```

Comment:
I'm confused as to how this auto rerenders the table when drizzle fetches a new set of pending requests. [This example](https://github.com/trufflesuite/drizzle-react#reading-state) uses the returned `drizzleState` variable. What's the difference between the example and this approach?

Reply:
Not sure. The example just below on the page you linked uses `useCacheCall`, which is the one I was following. I think this call to `useDrizzleState` is superfluous actually.